.devcontainer/devcontainer.json

```
@@ -0,0 +1,7 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+    },
+  "postCreateCommand": "python3 -m venv .venv & source .venv/bin/activate & pip3 install --upgrade pip & pip3 install -r requirements.txt",
+  "postStartCommand": "mkdocs serve"
+}
```

I would also add here the forwarded port

```suggestion
  "forwardPorts": [8000]
}
```